### DISCUSSION TITLE
Including servlet context path in GraphiQL

### DISCUSSION THREAD

**Discussion by debmitra** [Netflix/dgs-framework#208] [Q&A]
As one of my business requirements, I need to have GraphQL services exposed at servlet context path. Though I am able to invoke the service successfully with servlet context path e.g. http://localhost:8080/api/v1/mycontext/graphql, I am unable to see the project schema files when trying to access through GraphiQL URL e.g. http://loalhost:8080/api/v1/mycontext/graphiql or http://localhost:8080/graphiql.

Here are the versions I am using in my application:
Java: 1.8
Spring Boot: 2.3.6
Apache Maven: 3.6.1
graphql-java: 16.2
graphql-dgs-spring-boot-starter: 3.5.2

In order to address this issue, I think I need to have the graphql-dgs-spring-boot-starter dependency having the fix explained in pull request #175 . However, I am unable to get the dependency version through maven configuration. Here is the snippet of maven configuration I am using:
```
<dependency>
      <groupId>com.netflix.graphql.dgs</groupId>
      <artifactId>graphql-dgs-spring-boot-starter</artifactId>
      <version>v3.10.0-rc.1</version>
</dependency>
```

Please advise if this is the correct version and how do I get this fix in my application?

**Comment by paulbakker** [comment]
To get the RC versions you need to add an additional repository to your build config. For example in Gradle: 

```
repositories {
    mavenCentral()
    maven {
        url = "https://netflixoss.jfrog.io/artifactory/maven-oss-candidates/"
    }
}
```

I'll also add this to the docs for future reference.
